DOCUMENTATION.md

```diff
@@ -0,0 +1,305 @@
+# Reference API
+
+## HTML
+
+### Root Element
+
+impress.js requires a Root Element with id "impress". All the content of the presentation will be created inside that element. It is not recommended to manipulate any of the styles, attributes or classes that are created by impress.js inside the Root Element after initialization.
```

Comment:
@FagnerMartinsBrack I'm not sure how substeps connect to multiple impress instances.

Impress.js 'core' (the `impress()` factory function) supports multiple roots - meaning, you can create separate API objects for multiple root elements on the page, having multiple root elements initialised as impress presentations etc.

But of course global classes on body element may collide, styling and positioning may not work, etc. So even that it's technically possible I would probably not advertise having multiple roots on one page, because the result may be very unpredictable.

But for sure `impress("something")` can be used to initialise impress.js on element with different id than "impress" and it should be mentioned in docs.